common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4018,6 +4022,32 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("64m")
 
+  val CLIENT_PUSH_SORT_USE_ADAPTIVE_MEMORY_THRESHOLD: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.spark.push.sort.memory.useAdaptiveThreshold")
+      .withAlternative("celeborn.push.sortMemory.useAdaptiveThreshold")
+      .categories("client")
+      .doc("adaptively adjust threshold for sort shuffle writer's memory threshold")
```

Comment:
Keep the doc style that the first letter should be capitalized and a period should be added at the end.

Reply:
updated